tests/test_certification.py

```diff
@@ -31,3 +31,7 @@ def test_search_results(self):
             20.04+LTS&vendors=Lenovo"
         )
         self.assertEqual(response.status_code, 200)
+
+    def test_model_details(self):
+        response = self.client.get("/certification/201807-26311")
+        self.assertEqual(response.status_code, 200)
```

Comment:
Not a hard requirement, but it might be nice to check at least one piece of content on this page. Just to make sure we're not just sending a 200 for every `/certification/xxx` page or something silly.

Reply:
Do you mean checking the HTML?